packages/react-components/src/components/MeetingConferencePhoneInfo.tsx

```diff
@@ -0,0 +1,215 @@
+                </Stack.Item>
+                <Stack.Item className={phoneInfoStep}>
+                  {conferencePhoneInfoList.map((phoneNumber, index) => (
+                    <Stack.Item key={index}>
+                      <Text className={phoneInfoTextStyle}>{formatPhoneNumberInfo(phoneNumber, strings)}</Text>
+                    </Stack.Item>
+                  ))}
+                </Stack.Item>
+              </Stack>
+              <Stack
+                horizontal
+                horizontalAlign="space-between"
+                verticalAlign="center"
+                className={phoneInfoInctructionLine}
+              >
+                {}
+                <Stack.Item>
+                  <Stack horizontal>
+                    <Stack.Item className={phoneInfoIcon(theme)}>
+                      <Icon
+                        iconName="DtmfDialpadButton"
+                        style={{ color: theme.palette.themePrimary, padding: '8px' }}
+                      />
+                    </Stack.Item>
+                    <Stack.Item>
+                      <Text className={phoneInfoTextStyle}>{strings?.meetingConferencePhoneInfoModalMeetingId}</Text>
+                    </Stack.Item>
+                  </Stack>
+                </Stack.Item>
+                <Text className={phoneInfoTextStyle}>{conferencePhoneInfoList[0].conferenceId}#</Text>
+              </Stack>
+              <Stack horizontal horizontalAlign="space-between" verticalAlign="center">
+                <Stack horizontal>
+                  <Stack.Item className={phoneInfoIcon(theme)}>
+                    <Icon iconName="PhoneInfoWait" style={{ color: theme.palette.themePrimary, padding: '8px' }} />
+                  </Stack.Item>
+                  <Stack.Item>
+                    <Text className={phoneInfoTextStyle}>{strings?.meetingConferencePhoneInfoModalWait}</Text>
+                  </Stack.Item>
+                </Stack>
+              </Stack>
+            </Stack>
+          )}
+        </Modal>
+      }
+    </>
+  );
+};
+
+/**
+ * @internal
+ * format phone number
+ */
+export const formatPhoneNumber = (phoneNumber: string): string => {
+  const cleaned = ('' + phoneNumber).replace(/\D/g, '');
+  const match = cleaned.match(/^(\d{1,2})?(\d{3})(\d{3})(\d{4})$/);
+  if (match) {
+    const intlCode = '+' + match[1] + ' ';
+    return [intlCode, '(', match[2], ') ', match[3], '-', match[4]].join('');
+  }
+  return phoneNumber;
+};
+
+/**
+ * @internal
+ * format phone number
+ */
+export const formatPhoneNumberInfo = (
+  phoneNumber: _ConferencePhoneInfo | undefined,
+  strings: MeetingConferencePhoneInfoModalStrings | undefined
+): string => {
+  if (!phoneNumber) {
+    return '';
+  }
+  const toll = phoneNumber.isTollFree
+    ? strings?.meetingConferencePhoneInfoModalTollFree
+    : strings?.meetingConferencePhoneInfoModalToll;
+  const countryAndCity = [phoneNumber.country, phoneNumber.city].filter((x) => x).join(', ');
```

Comment:
Do all locales use comma `,` as the seperator between city and country?

Reply:
This is current Teams implementation:
+1 323-849-4874 United States, Los Angeles

@JamesBurnside Do we have in current implementation any text format (not translation) based on local?

Reply:
Resolving this comment thread as obsolete to: https://github.com/Azure/communication-ui-library/pull/4637/files#r1622592570. If we implement https://github.com/Azure/communication-ui-library/pull/4637/files#r1622592570 then the localization file takes care of any locale-specific delimeters